README.md

```diff
@@ -1,4 +1,59 @@
 Compose Switch 
 --------------
 
-This is a thin wrapper to actually run either docker-compose v1 (python) or docker compose v2 (compose CLI plugin) depending on user's configuration
+Compose Switch is a replacement to the Compose V1 `docker-compose` (python) executable. It translates the command line into Compose V2 `docker compose` then run the latter. 
+
+## installation
+
+We provide an script for automated installation:
+
+```console
+$ curl -fL https://raw.githubusercontent.com/docker/compose-cli/main/scripts/install/install_linux.sh | sh
+```
+
+### Manual installation
+
+1. download compose-switch binary for your architecture
+```console
+$ curl -fL https://github.com/docker/compose-switch/releases/download/v1.0.1/docker-compose-linux-amd64 -o /usr/local/bin/compose-switch
+```
+2. make compose-switch executable
+```console
+$ chmod +x /usr/local/bin/compose-switch
+```
+3. rename `docker-compose` binary if you already have it installed as `/usr/local/bin/docker-compose` 
+```console
+$ mv /usr/local/bin/docker-compose /usr/local/bin/docker-compose-v1
+```
+4. define an "alternatives" group for `docker-compose` command: 
+```console
+$ update-alternatives --install /usr/local/bin/docker-compose docker-compose <PATH_TO_DOCKER_COMPOSE_V1> 1
```

Comment:
If you installed docker-compose manually, use the path of the renamed file
If you installed install the docker-compose package from your distro, use the patch of docker-compose